pyfstat/core.py

```diff
@@ -1590,29 +1616,45 @@ def _init_semicoherent_window_range(self):
     def init_semicoherent_parameters(self):
         logging.info(
             (
-                "Initialising semicoherent parameters from {} to {} in" " {} segments"
+                "Initialising semicoherent parameters from"
+                " minStartTime={:d} to maxStartTime={:d} in {:d} segments..."
             ).format(self.minStartTime, self.maxStartTime, self.nsegs)
         )
-        self.transientWindowType = "rect"
-        self.whatToCompute = lalpulsar.FSTATQ_2F + lalpulsar.FSTATQ_ATOMS_PER_DET
```

Comment:
I think we need `lalpulsar.FSTATQ_ATOMS_PER_DET` to compute the semicoherent Fstat... I'll drop this over here so I don't forget.

Reply:
See [L1588](https://github.com/PyFstat/PyFstat/pull/126/files#diff-ed8f52637a7d535159fd46ef4d8a91a2R1588): when we set `self.computeAtoms` in `SemiCoherentSearch`, then the `ComputeFstat` it inherits from will automatically select the right `whatToCompute` ([L740](https://github.com/PyFstat/PyFstat/pull/126/files#diff-ed8f52637a7d535159fd46ef4d8a91a2R740))